inspirehep/modules/authors/views/holdingpen.py

```diff
@@ -280,10 +285,16 @@ def submitupdate():
     workflow_object.extra_data['is-update'] = True
     workflow_object.data = formdata_to_model(workflow_object, visitor.data)
     workflow_object.save()
+    engine = WorkflowEngine.with_name('author')
+    engine.save()
     db.session.commit()
 
     # Start workflow. delay will execute the workflow in the background
-    start.delay("author", object_id=workflow_object.id)
+    start.delay(
+        "author",
+        object_id=workflow_object.id,
+        engine_uuid_hex=engine.uuid.get_hex(),
+    )
```

Comment:
This is quite some boiler plate. Shouldn't it be better part of some utils or workflow method?